cores/esp32/esp32-hal-uart.c

```diff
@@ -513,12 +513,16 @@ uart_t *uartBegin(
     retCode &= ESP_OK == uart_param_config(uart_nr, &uart_config);
   }
 
-  // Is it right or the idea is to swap rx and tx pins?
-  if (retCode && inverted) {
-    // invert signal for both Rx and Tx
-    retCode &= ESP_OK == uart_set_line_inverse(uart_nr, UART_SIGNAL_TXD_INV | UART_SIGNAL_RXD_INV);
+  if (retCode) {
+    if (inverted) {
+      // invert signal for both Rx and Tx
+      retCode &= ESP_OK == uart_set_line_inverse(uart_nr, UART_SIGNAL_TXD_INV | UART_SIGNAL_RXD_INV);
+    } else {
+      // invert signal for both Rx and Tx
```

Comment:
Code comment should be updated if I get it right.
```suggestion
      // disable invert signal for both Rx and Tx
```